templates/hound.yml

```diff
@@ -0,0 +1,17 @@
+# See https://houndci.com/configuration for help.
+coffeescript:
+  # config_file: .coffeescript-style.json
+  enabled: false
```

Comment:
Suspenders creates a new project, so it seems unlikely that the plan would be to mix CS and JS. Suspenders has a solid history of providing a good default opinion. I think we should pick one or the other and run with that.